src/main/java/org/json/XMLParserConfiguration.java

```diff
@@ -303,4 +313,18 @@ public XMLParserConfiguration withForceList(final Set<String> forceList) {
     public XMLParserConfiguration withMaxNestingDepth(int maxNestingDepth) {
         return super.withMaxNestingDepth(maxNestingDepth);
     }
+
+    /**
+     * To enable explicit end tag with empty value.
+     * @param closeEmptyTag
+     * @return same instance of configuration with empty tag config updated
+     */
+    public XMLParserConfiguration withCloseEmptyTag(boolean closeEmptyTag){
+        this.closeEmptyTag = closeEmptyTag;
```

Comment:
Please follow the model of the other "with*" methods:
* Clone 'this' object
* Update the clone with the new property value
* Return the clone

Reply:
@stleary -updated code and test cases accordingly.